tools/raml2html/patches/php_7.4.patch

```diff
@@ -0,0 +1,11 @@
+--- /src/Body.php	2023-01-27 15:49:58
++++ /src/Body.php	2023-01-27 15:51:03
+@@ -228,7 +228,7 @@
+      */
+     public function getExample()
+     {
+-        return $this->examples[0];
++        return $this->examples[0] ?? '';
+     }
+
+     /**
```

Comment:
Looks like we have some trailing characters here and there's no EOL at the end of this file.

Reply:
EOL added in 9b726c0fdbb758949f5c2d8cee7f268f7e19bbdc